internal/backend/testing.go

```diff
@@ -424,7 +424,7 @@ func testLocksInWorkspace(t *testing.T, b1, b2 Backend, testForceUnlock bool, wo
 
 	_, err = lockerB.Lock(infoB)
 	if err == nil {
-		lockerA.Unlock(lockIDA)
+		lockerA.Unlock(lockIDA) //nolint:errcheck // test already failed, no need to check err further
```

Comment:
```suggestion
		_ = lockerA.Unlock(lockIDA)
```
I've had the same comment at another place: we can use Go's syntax here to ignore errors.